localflavor/mx/mx_states.py

```diff
@@ -9,11 +9,11 @@
     ('BCN', _('Baja California')),
     ('BCS', _('Baja California Sur')),
     ('CAM', _('Campeche')),
+    ('CDMX', _('Ciudad de México')),
```

Comment:
The state field is 3 characters long this won't work. You'll also need to modify the tests for this change.

Reply:
Also I believe it can be confusing with "Estado de México" which is a different state, I know its not very conventional but as a compromise could it be `CMX` ??